### DISCUSSION TITLE
Conversion of a folder to jpeg

### DISCUSSION THREAD

**Discussion by l1qu1d101** [ImageMagick/ImageMagick#6851] [Help]
### ImageMagick version

7.1.1-21-Q16-HDRI-x64-dll

### Operating system, version and so on

Windows 10 22H2

### Description

Is it possible to put together a command line script for conversion of a folder of images that I simply want to upload to steam at this point, they only accept jpegs yet my saved images are pngs mainly. (I have checked that IM is registered as a command line option and it is)

I'm also wanting to integrate the option as a right click context. I am using a program called openwithPP as an alternative to regedit (also on github)
![image](https://github.com/ImageMagick/ImageMagick/assets/16263017/f44df4d8-0c85-440f-9f38-44cf035e96c9)
added the image as reference for the options the app has.

it would be excellent to have this option in windows as standard with either Image Magick or Windows itself yet i'm struggling with the options, mainly since the help section for IM does not have specific instructions for conversion of images and also whether I could get IM to accept the images it is passed instead of specific images with specific locations. 

I would rlly appreciate some help with this, thanks to anyone who can.

**Comment by fmw42** [comment]
See `magick mogrify` at https://imagemagick.org/Usage/basics/#mogrify.   It allows you to convert a whole folder of images from one format to another.
